tests/src/test/scala/neotypes/AsyncSuite.scala

```diff
@@ -59,18 +57,3 @@ abstract class AsyncDriverProvider[F[_]](testkit: AsyncTestkit[F])
   override protected final def transact[T](driver: DriverType)(txF: TransactionType => F[T]): F[T] =
     driver.transact(txF)
 }
-
-/** Group all the Async specs into one big suite, which can be called for each Async type. */
-abstract class AsyncSuite[F[_]](testkit: AsyncTestkit[F])
```

Comment:
Some modules are not yet supported in Scala 3, so I added dedicated AsyncSuite for Scala 2 and Scala 3.